### PR TITLE
Set exit code on package build failure

### PR DESCRIPTION
**Pull request by anthonyserious**

### CODE REVIEW

**Comment by anthonyserious** [comment]
Resolves #29.

File: rpackage [hunk 1]
```diff
@@ -25,7 +25,7 @@ set_toolchain()
 function install_package() {
   pkg=$1
   ${R}script --vanilla \
-    -e "install.packages('$pkg', repos='$repo', dependencies=$dependencies, INSTALL_opts=$install_opts)"
+    -e "withCallingHandlers(install.packages('$pkg', repos='$repo', dependencies=$dependencies, INSTALL_opts=$install_opts), warning=function(w) quit('no', 1))"
```
Comment (muxspace) [inline rpackage]: Can you keep the line width < 80 chars? Other than that it looks good.

Comment (anthonyserious) [inline rpackage]: There you go.

**Comment by muxspace** [comment]
Wicked

**Pull request merged**